.devcontainer/devcontainer.json

```diff
@@ -53,9 +54,10 @@
 
 	// Use 'postCreateCommand' to run commands after the container is created.
 	// Installs:
-	//  	1. Mega Linter
-	//		2. Pester
-	"postCreateCommand": "npm install -g mega-linter-runner && pwsh -command Install-Module -Name Pester -Force -SkipPublisherCheck",
+	//   1. Mega Linter
+	//   2. Pester
+	//   3. Configures benchpress Python module
+	"postCreateCommand": "npm install -g mega-linter-runner && pwsh -command Install-Module -Name Pester -Force -SkipPublisherCheck && pip install --editable ./framework/python/",
```

Comment:
Should the mega-linter-runner and the pip install be included in the container Dockerfile?

Reply:
While this would be the best option, it's unfortunately not possible due to the order of execution which is:

1. Dockerfile
2. Installation of *features* defined in *devcontainer.json*. Here we install the following:
  - Azure CLI
  - Powershell
  - Git
  - Node
  - Python
 3. At start-up of container, install:
  - Mega-linter using NPM
  - Pester using Powershell
  - Our *benchpress* Python module using PIP

The downside of this is also that every time the container spawns, the three packages/modules (MegaLinter, Pester, and Benchpress) need to be installed and add some time to start up. Whereas with a Docker container, these would be baked into the image.

Open to suggestions if there are better ways to build this. Also as a note, using *features* in dev container was the best option I found for installing different languages.